libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -340,6 +340,16 @@ export const parentalLeaveFormMessages: MessageDir = {
       description:
         'In order to apply for this application we need your email and phone number',
     },
+    familyInformationTitle: {
+      id: 'pl.application:familyinformation.title',
+      defaultMessage: 'Upplýsingar um fjölskyldumeðlimi',
+      description: 'Your family information',
```

Comment:
Just for a reminder, the description is not used to pass the English translation, but more for the translations people to know what the message it is referring to. The description won't be used to populate the Contentful English field but just used to be displayed as the "description" section